docs/quickstarts/excel-quickstart-vue.md

```diff
@@ -67,16 +67,29 @@ After you complete the wizard, it creates a `my-office-add-in` folder, which con
 
 [!include[HTTPS guidance](../includes/https-guidance.md)]
 
-To enable HTTPS for your app, create a `vue.config.js` file in the root folder of the Vue project with the following contents:
+1. To enable HTTPS for your app, create a `vue.config.js` file in the root folder of the Vue project with the following contents:
+
+    ```js
+    var fs = require("fs");
+    var path = require("path");
+    var homedir = require('os').homedir()
+    
+    module.exports = {
+        devServer: {
+            port: 3000,
+            https: true,
+            key: fs.readFileSync(path.resolve(`${homedir}/.office-addin-dev-certs/localhost.key`)),
```

Comment:
Users will have to run "npx office-addin-dev-certs install" manually to have this. The certs expire after 30 days and so it would need to be re-run. 

It would be even better if the office-addin-dev-certs package is a dev dependency and the code updated to use the office-addin-dev-certs package and call the getHttpsServerOptions() function. This is an async function, and I'm not familiar with Vue to know if it supports resolving the promise (e.g. using await) before returning the module exports.

That would be better because this call will create new certs if the certs are missing/expired.

Reply:
Also, for webpack, the key/cert/ca properties are in an https object for devServer. This https object is what getHttpsServerOptions returns. Just wanted to mention because the vue config may be different but this appears incorrect to me, although Courtney said that it did work.

Reply:
I was able to get some tester code for await/async working. If you could provide the actual code for `getHttpsServerOptions`, I can plug that in.

```
var fs = require("fs");
var path = require("path");
var homedir = require('os').homedir();

test();

module.exports = {
    devServer: {
        port: 3000,
        https: true,
        key: fs.readFileSync(path.resolve(`${homedir}/.office-addin-dev-certs/localhost.key`)),
        cert: fs.readFileSync(path.resolve(`${homedir}/.office-addin-dev-certs/localhost.crt`)),
        ca: fs.readFileSync(path.resolve(`${homedir}/.office-addin-dev-certs/ca.crt`))
    }
};

async function test(){
    await test2();
}

async function test2(){
    return 2;
}
```

Reply:
It would be something like:

```
const devCerts = require("office-addin-dev-certs");

const httpsServerOptions = await devCerts.getHttpsServerOptions();
module.exports = {
   devServer: {
      https: httpsServerOptions;
   }
}
```

Or if you need the key/cert/ca properties under devServer, you can assign them as:

```
devServer: {
    https: true;
    key: httpsServerOptions.key;
    ca: httpsServerOptions.ca;
    cert: httpsServerOptions.cert;
}
```

Reply:
You can use `npx office-addin-dev-certs uninstall` to be able to remove the certs and test out the experience for the user when running and no certs are installed.

There will be a dialog prompt on Windows or a password prompt for Mac.

Once the certs are already valid, there will be no prompt.

Reply:
This still has the `await` problem. 

When removing the `await`, I end up with the same security error. My setting of the port to 3000 is also being ignored.

```
    var fs = require("fs");
    var path = require("path");
    var homedir = require('os').homedir()
  
    var devCerts = require("office-addin-dev-certs");

    var httpsServerOptions = devCerts.getHttpsServerOptions();
    module.exports = {
      devServer: {
        port: 3000,
        https: true,
        key: httpsServerOptions.key,
        ca: httpsServerOptions.ca,
        cert: httpsServerOptions.cert
      }
    }
```

![image](https://user-images.githubusercontent.com/38896772/79138631-b0d4de00-7d69-11ea-9671-910e3bf7ccf4.png)

Reply:
Yeah without the await, devCerts.getHttpsServerOptions(); will return pending and thus the port and other options won't be defined.  Let me see if I can figure out how we can call devCerts.getHttpsServerOptions();

Reply:
As far as I can tell, vue.config.js doesn't support async, at least not out of the box.  Perhaps this is doable but I am not sure how.  My proposed solution is synchronous and works so maybe we should just go with that

Reply:
Thanks @TCourtneyOwen. I applied you suggested changes to vue.config.js from the other thread. Does this PR match what you intended?